samples/basic/attack/read_memory.sh

```diff
@@ -21,7 +21,8 @@ sudo chown "$(id -u -n):$(id -g -n)" "$mem_file"
 
 echo "Searching memory for string \"${search_string}\" in \"${mem_file}\"..."
 
-if (strings "${mem_file}" | grep -i "${search_string}"); then
+if grep -Fxq "${search_string}" "${mem_file}"
+then
```

Comment:
I believe our standard is always "; then" for bash if/then formatting. Based on what I've seen in the project.

Reply:
Done.